### PR TITLE
First version

### PR DESCRIPTION
**Pull request by elmerzsolt**

### CODE REVIEW

File: css/components.css [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+ *
+ */
+
+.contact-container {
+  padding:          4.286rem;
+  border-radius:    1.071rem;
+  background-color: #ffffff;
+}
+
+  .contact-element { margin-bottom: 2.143rem; }
```
Comment (freegyes) [inline css/components.css:131]: fix indent

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          <a class="navigation__link">Contact</a>
+        </nav>
+      </div>
+    </div>
+  </header>
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply index.html:47]: These are coming from these:
https://google.github.io/styleguide/htmlcssguide.html
https://cssguidelin.es/

We agreed with Vadas to use this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  <div class="layout__item--70-percent layout__item--centered-horizontally gutter--horizontal-big clearfix">
+
+
+    <div class="contact-container layout__item--50-percent-w-gutter layout__item--floated-left">
+
+
+        <div class="contact-element">
+          <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3304.4556889126807!2d-118.341063884784!3d34.08346498059731!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x80c2b8d36cb40581%3A0xe92d0cfe124170cf!2s3481%20Melrose%20Ave%2C%20Los%20Angeles%2C%20CA%2090038%2C%20Egyes%C3%BClt%20%C3%81llamok!5e0!3m2!1shu!2shu!4v1586337676963!5m2!1shu!2shu"
+                  width="600" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false"
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply index.html:56]: Okay, I changed it, here I just used the copied basic code that google gave me.

File: css/base.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  font-weight:    bold;
+  text-transform: uppercase;
+  letter-spacing: .171rem;
+}
+
+iframe {
+  width:          100%;
+  height:         20rem;
+  border-radius:  1.071rem;
+}
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/base.css:96]: Okay, I moved it to the settings.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>Contact Page</title>
+  <meta name="description" content="">
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply index.html:7]: Added.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="stylesheet" href="css/normalize.min.css">
+  <link rel="stylesheet" href="css/base.css">
+  <link rel="stylesheet" href="css/layout.css">
+  <link rel="stylesheet" href="css/components.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap">
+
+<!--  //TODO: Get code for FA5, this is using w3school.com's now.-->
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply index.html:17]: Changed it.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+      <div class="form-element">
+        <h2>Contact Us</h2>
+        <p class="form__summary">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+      </div>
+
+
+      <div class="form-element">
+        <input type="text" placeholder="Name" class="form__input-field"><br>
+        <input type="text" placeholder="Enter Email" class="form__input-field"><br>
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply index.html:99]: I'm not sure what you are reffering as "this way", can you please explain it in a more specific way?

File: css/base.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+*    { box-sizing: border-box;} /* [1] */
+html { font-size:  14px; } /* [2] */
+
+
+body {
+  font-family:      'Montserrat', sans-serif;
+  color:            #211138;
+  background-color: #f4f3f5;
+}
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/base.css:55]: Moved it there.

File: css/base.css [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+\*------------------------------------*/
+
+h2 { font-size: 2.857rem; }
+
+h6,
+button,
+a {
+  font-size:      .857rem;
+  font-weight:    bold;
+  text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/base.css:88]: Okay, I change it, but again, it should be discussed with Vadas as well.

File: css/components.css [hunk 10]
```diff
@@ -0,0 +1,232 @@
...
+  text-align:     center;
+}
+
+  .button--primary-solid {
+    border-color:     #6200ea;
+    color:            #ffffff;
+    background-color: #6200ea;
+  }
+
+  .button--secondary-solid:visited { color: #ffffff; }
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/components.css:52]: Yes, this is a mistake, changed it.

File: css/components.css [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+\*------------------------------------*/
+
+.button {
+  display:        inline-block;
+  border:         2px solid;
+  border-radius:  1.714rem;
+  text-align:     center;
+}
+
+  .button--primary-solid {
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/components.css:46]: All of these intendation rules should be discussed with Vadas.

File: css/components.css [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+/*------------------------------------*\
+  #FEJLÉC
+\*------------------------------------*/
+
+.header {
+  height:           80px;
+  background-color: #6200ea;
+}
+
+  .logo {
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/components.css:67]: All of these intendation rules should be discussed with Vadas.

File: css/components.css [hunk 13]
```diff
@@ -0,0 +1,232 @@
...
+\*------------------------------------*/
+/**
+ * 1. Ez a padding ahhoz kell, hogy a hover effektus fehér sávja a megfelelő
+ * helyen jelenjen meg.
+ *
+ * 2. Ezzel a szabállyal vesszük le az első linkről a felesleges bal oldali
+ * margint.
+ */
+
+.navigation { height: 80px; }
```
[the remark this answers is not in the document]
Comment (elmerzsolt) [reply css/components.css:87]: All of these intendation rules should be discussed with Vadas.

File: css/components.css (hunk 1 above)
Comment (elmerzsolt) [reply css/components.css:131]: All of these intendation rules should be discussed with Vadas.

**Pull request merged**